README.md

```diff
@@ -105,13 +128,14 @@ ALTER TABLE <table> SET TBLPROPERTIES("beekeeper.remove.expired.data"="true", "b
 
 # Running Beekeeper
 
-Beekeeper consists of three Spring Boot applications which run independently of each other:
+Beekeeper consists of four Spring Boot applications which run independently of each other:
```

Comment:
This could get simplified by saying that all Beekeeper modules are run independently (without mentioning how many are available). Mainly, to avoid having to update the number every time there's a new module.